docs/resources/volume.md

```diff
@@ -57,17 +56,17 @@ resource "databricks_volume" "this" {
 
 The following arguments are supported:
 
-* `name` - Name of volume
-* `catalog_name` - Name of parent catalog
+* `name` - Name of the Volume
+* `catalog_name` - Name of parent Catalog
 * `schema_name` - Name of parent Schema relative to parent Catalog
-* `volume_type` - URL of storage location. Currently only `EXTERNAL` is supported
-* `owner` - (Optional) Name of the volume owner
-* `storage_location` - (Optional) If `EXTERNAL` volume type is used, then location of that volume
-* `comment` - (Optional) User-supplied free-form text.
+* `volume_type` - Volume type. `EXTERNAL` or `MANAGED`.
+* `owner` - (Optional) Name of the volume owner. Can only be used for Update.
```

Comment:
we can't setup owner on the creation?

Reply:
```suggestion
* `owner` - (Optional) Name of the volume owner. 
```
this is irrelevant and incorrect: https://github.com/databricks/terraform-provider-databricks/blob/master/catalog/resource_volume.go#L51-L63